version/version.go

```diff
@@ -1,7 +1,7 @@
 package version
 
 const Maj = "0"
-const Min = "8" // validator set changes, tmsp->abci, app persistence/recovery, BFT-liveness fix
```

Comment:
Is this the best place for such comments? maybe we should create `CHANGELOG.md` instead?